modules/meb_api/spec/requests/meb_api/v0/education_benefits_spec.rb

```diff
@@ -20,10 +31,15 @@
   end
 
   describe 'GET /meb_api/v0/eligibility' do
-    it 'successfully returns JSON' do
-      get '/meb_api/v0/eligibility'
-      expect(response).to have_http_status(:ok)
-      JSON.parse(response.body).to_yaml
+    context 'Veteran who has benefit eligibility' do
+      it 'returns a 200 with eligibility data' do
+        VCR.use_cassette('dgi/get_eligibility') do
+          get '/meb_api/v0/eligibility'
+          expect(response).to have_http_status(:ok)
+          expect(response.body).to be_a(String)
+          # expect(response).to match_response_schema('dgi/eligibility_response', { strict: false })
```

Comment:
Is line 40 meant to be commented out here?

Reply:
It was intentionally commented out as it was causing the test suite to fail. I'm open to feedback on getting the match_response_schema method working.

Reply:
What were the errors?

Reply:
`     Failure/Error: JSON::Validator.validate!(schema_path.to_s, json, { strict: true }.merge(opts))

     JSON::Schema::JsonParseError:
       expected hash key, not a hash close (after properties.data.properties.type.type.properties.chapter) at line 32, column 16 [par
     # ./spec/support/matchers/schema_matchers.rb:10:in `valid_against_schema?'`

Reply:
Ok, I feel like the schema match is still a valid test to have passing though.

Reply:
Agreed. Does that error give you any idea what the root cause may be?

Reply:
Does [this](https://stackoverflow.com/questions/7543779/ruby-json-parse-changes-hash-keys) maybe help?

Reply:
I traced it back to the schema file itself. Should be addressed in the latest commit on this branch.